src/components/modal/ModalCreateBooks.tsx

```diff
@@ -0,0 +1,188 @@
+    AuthorService.getAuthors().then((response) => {
+      setAtuhors(response)
+    })
+    setShowAddAuthor(true)
+    setShowAddButton(true)
+  }
+
+  const handlerCloseModal = () => {
+    onClose()
+  }
+
+  const handleFormAuthor = () => {
+    setShowAddAuthor(false)
+    setShowAddButton(false)
+  }
+
+  const handleCloseFormAuthor = () => {
+    setShowAddAuthor(true)
+    setShowAddButton(true)
+  }
+
+  useEffect(() => {
+    AuthorService.getAuthors()
+      .then(response => {
+        setAtuhors(response)
+      })
+  }, [])
+
+  const handleAuthor = (newAuthors: MultiValue<Author>) => {
+    setBooksData((prev) => ({ ...prev, AuthorIds: newAuthors as Author[] }))
+  }
+
+  if(!open) return null
+
+
+  return ReactDOM.createPortal(
+    <div className='overlay'>
+      <div className='content'>
+        <form className='form_for_modal' onSubmit={hanlderCreateBooks}>
+          <div className='heder_modal'>
+            <button type='button' onClick={handlerCloseModal} className='x_button_for_modal'><XIconForModal className='x_icon_for_modal'/></button>
+          </div>
+          <img className='modal_cover_img' src={coverImg ? coverImg : placeholder} />
+          <div className='container_for_element'>
+            <input type='text' className='input_for_title' placeholder='Title' onChange={({ target }) => setBooksData((prevTitle) => ({ ...prevTitle, Title: target.value }))}/>
+          </div>
+          <div className='container_for_element'>
+            <input type='text' className='input_for_description' placeholder='Description' onChange={({ target }) => setBooksData((prevDescription) => ({ ...prevDescription, Description: target.value }))}/>
+          </div>
+          <div className='container_for_element'>
+            <input type='text' className='input_for_isbn' placeholder='ISBN' pattern='^(?=(?:\D*\d){10}(?:(?:\D*\d){3})?$)[\d-]+$' onChange={({ target }) => setBooksData((prevIsbn) => ({ ...prevIsbn, Isbn: target.value }))}/>
+          </div>
+          <div className='container_for_element'>
+            <input type='text' className='input_for_quantity' placeholder='Quantity' onChange={({ target }) => setBooksData((prevQuantity) => ({ ...prevQuantity, Quantity: +target.value }))}/>
+          </div>
+          <div className='container_for_element'>
+            <input type='date' className='input_for_date' onChange={({ target }) => setBooksData((prevDate) => ({ ...prevDate, PublishDate: target.value }))}/>
+          </div>
+          <div className='container_for_element'>
+            <input type='file' className='input_for_image' onChange={handleImgUpload}/>
+          </div>
+          <div className='container_for_element'>
+            <Select name='authors' id='authors' className='select_for_authors' getOptionLabel={(author : Author) => `${author.FirstName} ${author.LastName}`} isMulti={true} getOptionValue={(option:Author) => option.Id.toString()} options={authors} onChange={handleAuthor} />
```

Comment:
You can write isMulti, no need for ={true} for boolean values

Reply:
I fix this.